.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -389,7 +389,7 @@ def write(x=''):
 
     with open(DATABASE_JSON, 'w') as json_file:
         json.dump(config, json_file, indent=2)
-    shutil.chown(DATABASE_JSON, HWPHP_USER, 'www-data')
+    shutil.chown(DATABASE_JSON, 'www-data', HWCRONPHP_GROUP)
```

Comment:
we're getting rid of www-data in a later issue/PR, right?

Reply:
Yes, in #1904. Going to leave it for now till that is done (so as to not break git)